octoprint_mrbeam/static/js/laserhead_changed.js

```diff
@@ -0,0 +1,60 @@
+$(function () {
+    function LaserheadChangedViewModel(params) {
```

Comment:
I think you will need to argument for why there is a separate viewmodel for this feature.

Reply:
Well that's the way we are handling this for now (check focus reminder). But I agree we need to revamp this at some point and have all such windows belong to one View model.